mods/cnc/rules/world.yaml

```diff
@@ -271,6 +271,7 @@ World:
 		PreviewActor: fact.colorpicker
 		PresetHues: 0, 0.125, 0.185, 0.4, 0.54, 0.66, 0.79, 0.875, 0, 0.14, 0.23, 0.43, 0.54, 0.625, 0.77, 0.85
 		PresetSaturations: 0.9, 0.9, 0.9, 0.9, 0.9, 0.9, 0.9, 0.9, 0.4, 0.5, 0.4, 0.5, 0.4, 0.5, 0.4, 0.5
+		PresetValues: 0.95, 0.95, 0.95, 0.95, 0.95, 0.95, 0.95, 0.95, 0.95, 0.95, 0.95, 0.95, 0.95, 0.95, 0.95, 0.95
```

Comment:
As mentioned in Discord, please change these to hex notation.

Can we please also replace the second row of desaturated colours in each mod with darkened colours instead?

Reply:
The former would be for a follow up PR